content/200-concepts/100-components/02-prisma-client/058-transactions.mdx

```diff
@@ -63,6 +74,13 @@ The following bulk operations run as transactions:
 
 ## The <inlinecode>$transaction</inlinecode> API
 
+The `$transaction` API can be used in two ways:
+
+- sequential operations: pass an array of Prisma Client operations that will be executed sequentially
+- interactive transactions (in Preview): pass a function in which you can perform a set of (interactive) transactions
```

Comment:
This can probably reuse result from https://github.com/prisma/docs/pull/2694/files#r783311720 - waiting for that to get merged and refined before making suggestion to adopt here as well.

Reply:
Done